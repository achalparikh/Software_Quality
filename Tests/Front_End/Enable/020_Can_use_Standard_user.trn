10                      00000 00000000 A 
09 John Doe             00012 00000000   
00                      00000 00000000   
10 John Doe             00000 00000000 S 
04 John Doe             00012 00500.00 S 
00                      00000 00000000   