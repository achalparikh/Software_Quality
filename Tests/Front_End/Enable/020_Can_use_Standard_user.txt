login
admin
enable
John Doe
00014
logout
login
standard
John Doe
deposit
00014
500.00
logout