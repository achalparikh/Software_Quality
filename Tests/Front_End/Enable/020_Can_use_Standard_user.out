Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as an admin
Enter a command:
Enter an account holder:
Enter an account number belonging to the account holder:
Account "00012" enabled
Enter a command:
You are now logged out
Login to start:
Enter type of account: "standard" or "admin"
Logging in as standard user - enter name:
You are now logged in as "John Doe"
Enter a command:
Enter an account number belongin to the account holder:
Deposited $500.00 to account "00012"
Enter a command:
You are now logged out
